core/templates/components/shared-component.module.ts

```diff
@@ -33,10 +34,14 @@ import { MaterialModule } from './material.module';
 import { TranslatePipe } from 'filters/translate.pipe';
 import { SkillMasteryViewerComponent } from
   './skill-mastery/skill-mastery.component';
-import { SharingLinksComponent } from
-  './common-layout-directives/common-elements/sharing-links.component';
 import { ExplorationEmbedButtonModalComponent } from
   './button-directives/exploration-embed-button-modal.component';
+import { SharingLinksComponent } from
+  './common-layout-directives/common-elements/sharing-links.component';
```

Comment:
Hey @srijanreddy98 Why are we not using absolute imports here and below?

Reply:
These imports are from this module (i.e. can be also considered from within the directory itself), so relative imports makes more sense here than absolute.

Reply:
Okay. Let's add this to the wiki if not already, just to avoid confusion when to use relative vs absolute imports.